02 Algorithm Reference/14 Reality Modelling/06 Margin Models.html

```diff
@@ -5,7 +5,7 @@
 Margin models control how much buying power (leverage) your algorithm has to make trades. Margin calculations can be very complex and depend on many factors including the brokerage or even time of day.
 </p>
 <p>
-	Margin models implement the $[ISecurityMarginModel] interface and default to the $[SecurityMarginModel] class. We also provide the $[PatternDayTradingMarginModel] to model intraday pattern day trading for US equities which provides 4x intraday leverage and 2x overnight leverage.
+	Margin models implement the $[ISecurityMarginModel] interface and default to the $[SecurityMarginModel,T:QuantConnect.Securities.SecurityMarginModel] class. We also provide the $[PatternDayTradingMarginModel,T:QuantConnect.Securities.PatternDayTradingMarginModel] to model intraday pattern day trading for US equities which provides 4x intraday leverage and 2x overnight leverage.
```

Comment:
`ISecurityMarginModel` was renamed to `IBuyingPowerModel`:
`$[IBuyingPowerModel, T:QuantConnect.Securities.IBuyingPowerModel]`